lib/config-generator.js

```diff
@@ -155,67 +117,128 @@ export class ConfigGenerator {
+                    ? "  { files: [\"**/*.jsonc\"], plugins: { json }, language: \"json/jsonc\" },\n"
+                    : "  { files: [\"**/*.jsonc\"], language: \"json/jsonc\", extends: [\"json/recommended\"] },\n";
+
+                exportContent += config;
+            }
+            if (languages.includes("json5")) {
+                const config = purpose === "syntax"
+                    ? "  { files: [\"**/*.json5\"], plugins: { json }, language: \"json/json5\" },\n"
+                    : "  { files: [\"**/*.json5\"], language: \"json/json5\", extends: [\"json/recommended\"] },\n";
+
+                exportContent += config;
+            }
         }
 
-        if (this.answers.framework === "vue") {
-            this.result.devDependencies.push("eslint-plugin-vue");
-            importContent += "import pluginVue from \"eslint-plugin-vue\";\n";
-            exportContent += "  pluginVue.configs[\"flat/essential\"],\n";
+        // language = markdown
+        if (languages.includes("md")) {
+            this.result.devDependencies.push("@eslint/markdown");
+            importContent += "import markdown from \"@eslint/markdown\";\n";
+
+            if (purpose === "syntax") {
+                const config = this.answers.mdType === "commonmark"
+                    ? "  { files: [\"**/*.md\"], plugins: {markdown}, language: \"markdown/commonmark\" },\n"
+                    : "  { files: [\"**/*.md\"], plugins: {markdown}, language: \"markdown/gfm\" },\n";
+
+                exportContent += config;
+            } else if (purpose === "problems") {
+                exportContent += "  extends: [\"markdown/recommended\"],\n";
 
-            // https://eslint.vuejs.org/user-guide/#how-to-use-a-custom-parser
-            if (this.answers.language === "typescript") {
-                exportContent += "  { files: [\"**/*.vue\"], languageOptions: { parserOptions: { parser: tseslint.parser } } },\n";
+                if (this.answers.mdType === "gfm") {
+
+                    // the default is commonmark
+                    exportContent += "  { files: [\"**/*.md\"], language: \"markdown/gfm\" },\n";
+                }
             }
         }
 
-        if (this.answers.framework === "react") {
-            this.result.devDependencies.push("eslint-plugin-react");
-            importContent += "import pluginReact from \"eslint-plugin-react\";\n";
-            exportContent += "  pluginReact.configs.flat.recommended,\n";
+        if (languages.includes("css")) {
+            this.result.devDependencies.push("@eslint/css");
+            importContent += "import css from \"@eslint/css\";\n";
+
+            if (purpose === "syntax") {
+                exportContent += "  { files: [\"**/*.css\"], plugins: { css }, language: \"css/css\" },\n";
+            } else if (purpose === "problems") {
+                exportContent += "  { files: [\"**/*.css\"], language: \"css/css\", extends: [\"css/recommended\"] },\n";
```

Comment:
```suggestion
                exportContent += "  { files: [\"**/*.css\"], plugins: { css }, language: \"css/css\", extends: [\"css/recommended\"] },\n";
```

We need to add the plugin here.